src/features/auth/components/AuthForm/AuthForm.css

```diff
@@ -14,34 +14,33 @@
   position: absolute;
   left: 50%;
   bottom: calc(100% - 4px);
-  width: 142px;
-  height: 81px;
+  width: 147px;
+  height: 85px;
   transform: translateX(-50%);
 }
 
 .auth-form__form {
   position: relative;
   margin: auto;
   border-radius: 10px;
-  padding: 24px 16px 76px;
+  padding: 24px 34px 60px;
   width: 100%;
   box-shadow: 1px 4px 25px rgba(95, 108, 114, 0.2);
   background-color: #fff;
 }
 
 .auth-form__messages-box {
-  margin-bottom: 36px;
-  min-height: 30px;
+  margin-bottom: 16px;
+  min-height: 20px;
 }
 
 .auth-form__message {
-  border-radius: 5px;
   padding: 7px 16px;
```

Comment:
there is no need for padding now
just need `margin-bottom` for all except last child

and dont forget to adjust another form paddings after this fix